cmd/lambda/accounts/list.go

```diff
@@ -1,32 +1,26 @@
 package main
 
 import (
-	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/Optum/dce/pkg/api"
+	"github.com/Optum/dce/pkg/errors"
 	"log"
 	"net/http"
 
 	"github.com/Optum/dce/pkg/api/response"
-	"github.com/Optum/dce/pkg/db"
-	"github.com/aws/aws-lambda-go/events"
 )
 
-type listController struct {
-	Dao db.DBer
-}
-
-// Call handles GET /accounts requests. Returns a response object with a serialized list of accounts.
-func (controller listController) Call(ctx context.Context, req *events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
```

Comment:
This was some old controller code for listing accounts, that wasn't actually used anywhere.

Moved the `GetAllAccounts` method from `get.go` into this file.